script.js

```diff
@@ -15,4 +15,28 @@ function generateGrid(pixles)
         item.addEventListener('mouseover', () => { item.style.backgroundColor = 'black'; });
     })
 }
-generateGrid(16);
\ No newline at end of file
+generateGrid(16);
+//reset button
+function clearGrid() {
+    document.querySelectorAll('.grid-item').forEach(item => item.style = '')
+  };
+  document.querySelector(".reset-btn").addEventListener('click', clearGrid);
+
+// New grid button
+document.querySelector(".grid-btn").addEventListener("click",newGrid)
+function newGrid() {
+       
+        let differentGridSize = prompt("Enter a new grid size(1-100):");
```

Comment:
Thank you so much Nujood  for making me pay attention to this !  Because I deeply know what does the button know, I forgot that the user does not know the exact code